crates/bevy_core_pipeline/Cargo.toml

```diff
@@ -28,4 +28,4 @@ bevy_transform = { path = "../bevy_transform", version = "0.8.0-dev" }
 bevy_utils = { path = "../bevy_utils", version = "0.8.0-dev" }
 
 serde = { version = "1", features = ["derive"] }
-
+radsort = "0.1"
```

Comment:
I'd like us to choose what turns out to be the best option from `radsort`, `voracious`, and `rdst` with consideration that we would like to be able to configure single-threaded sorting as discussed in #4291

Reply:
See results below.